lib/internal/Magento/Framework/App/Router/ActionList.php

```diff
@@ -34,7 +34,7 @@ class ActionList
         'for', 'foreach', 'function', 'global', 'goto', 'if', 'implements', 'include', 'instanceof',
         'insteadof','interface', 'isset', 'list', 'namespace', 'new', 'or', 'print', 'private', 'protected',
         'public', 'require', 'return', 'static', 'switch', 'throw', 'trait', 'try', 'unset', 'use', 'var',
-        'while', 'xor',
+        'while', 'xor', 'void',
```

Comment:
@antonkril why do we even need such check instead of simple fallback to a `VoidAction` in case `Void` class is not available?